[TUGAS_SELEKSI_2_13514089_13514043]

ramosjanoah opened the pull request:
Uploading pertama tugas 2.

ramosjanoah committed:
Add files via upload

Uploading pertama tugas 2.

ramosjanoah commented:
Uploading kedua tugas 2.
Ada kesalahan pada penguploadan tugas pertama.

ramosjanoah committed:
Uploading Tugas 2

Uploading Tugas 2

albertlieyingadrian commented:
@ramosjanoah Bisakah ditambahkan 1 file untuk langkah-langkah eksekusi keseluruhan project baik untuk tahap analisis maupun visualisasi ? Terima kasih.

ramosjanoah committed:
Delete Data_Hasil_Analisis.xlsx

ramosjanoah commented:
@LieAlbertTriAdrian Dalam format apakah file tersebut? .txt atau .doc?

albertlieyingadrian commented:
@ramosjanoah txt saja

ramosjanoah committed:
Upload Langkah-langkah dan Revisi Laporan

- Menambahkan file .txt yang berisi langkah-langkah pengerjaan laporan
- Menambahkan laporan dengan ekstensi .pdf (sebelumnya masih .word)

ramosjanoah committed:
Melakukan delete pada laporan .doc

ramosjanoah commented:
@LieAlbertTriAdrian Langkah-langkah pembuatan sudah saya buat. Laporan juga sudah saya buat dalam bentuk .pdf. Mohon maaf atas kesalahan sebelumnya. Mohon beri tahu lebih lanjut jika ada kesalahan lainnya. Terimakasih.
